common/prompt.ts

```diff
@@ -365,7 +361,7 @@ export function getPromptParts(opts: PromptPartsOptions, lines: string[], encode
   }
 
   parts.sampleChat = (
-    replyAs._id === char._id && chat.useOverrides
+    replyAs._id === char._id && !!chat.overrides
```

Comment:
presence of a persona override dictate whether the scenario and sample chat override is applied. i think i understand why, but the logic behind it is hard to understand. if i get it right, this should explain it:
```ts
// if user disabled overrides, every override will be empty, including chat.overrides
// (misleading name for the persona override). so if chat.overrides is absent, we can
// assume all overrides are disabled. if it's present, we can assume all overrides are enabled.
```
the `useOverrides` field was easier to understand imo...

and what if the user's override is to completely remove the persona field down to an empty string? the app will think greeting/scenario/samplechat overrides are disabled.

Reply:
I agree `useOverride` is better, but it wasn't backwards compatible :(
At some point (tm) hopefully we can circle back and centralise some logic.